privacyidea/lib/eventhandler/usernotification.py

```diff
@@ -345,7 +351,20 @@ def do(self, action, options=None):
                 mimetype = handler_options.get("mimetype", "plain")
                 useremail = recipient.get("email")
                 reply_to = handler_options.get("reply_to")
-
+                attachment = handler_options.get("attachment", False)
+
+                if attachment and googleurl_img:
+                    # get the image part of the googleurl
+                    googleurl = urlopen(googleurl_img)
+                    assert googleurl.headers.get('Content-Type') == 'image/png'
```

Comment:
We always create the googleurl_img to be a png.
So what would be the assert for?

Reply:
Was a check for me that the URL parsing worked as expected. I'll remove it.

Reply:
Done in bb386797